libvast/test/expression_evaluation.cpp

```diff
@@ -153,4 +156,23 @@ TEST(evaluation - schema) {
   CHECK(caf::holds_alternative<caf::none_t>(*ast_resolved));
 }
 
+TEST(evaluation - table slice rows) {
+  // Calling the fixture ctor makes sure the slices are available.
+  fixtures::events dummy;
```

Comment:
Isn't the ctor called upon test entry? The comment unfortunately doesn't help me understand what's going on.

Reply:
Only if the fixture actually inherits from `fixtures::events`. I did that instead to make the dummies disappear.